includes/wc-deprecated-functions.php

```diff
@@ -14,6 +14,12 @@
 	exit;
 }
 
+include_once( 'abstracts/abstract-wc-deprecated-hooks.php' );
+include_once( 'class-wc-deprecated-action-hooks.php' );
```

Comment:
Other than that, the logic of the deprecated filter classes looks solid to me. You can make a new issue if you want me to write some tests for these classes to verify.